examples/prismic/app/blog/[slug]/page.tsx

```diff
@@ -0,0 +1,50 @@
+import { notFound } from 'next/navigation'
+
+import { MakeswiftComponent } from '@makeswift/runtime/next'
+import { getSiteVersion } from '@makeswift/runtime/next/server'
+
+import { BLOG_POST_EMBEDDED_COMPONENT_ID } from '@/components/Builtins/BlogPost/BlogPost.makeswift'
```

Comment:
🍹 What if instead of exporting the component ID, we exported a server component from `BlogPost.makeswift.ts` that rendered `<MakeswiftComponent>` with the appropriate type applied and fetched its own `snapshot`? We'd only have to pass in the ID